lib/annealing/metal.rb

```diff
@@ -27,14 +27,21 @@ def cool!(new_temperature)
       end
     end
 
+    # True if cooled_metal.energy is lower than current energy.
```

Comment:
```suggestion
```

Reply:
Also if you create a new function, it should be bellow the prefer? function so it turns easy to read